src/shared/ui/section-label/section-label.module.scss

```diff
@@ -1,5 +1,5 @@
-.section-label {
-  margin: 8px;
+.label {
+  margin: 8px 8px 8px 0;
```

Comment:
Let's make it 40px like in layout, but for main-course banners we can define it 8px as it is now on the site

Reply:
I agree with you, it was with margin variations that the first refactoring commit was made, our teamlead asked to return everything as it was. And without his consent, I think it’s not worth changing this...

Reply:
Decided to leave it at 8px.